buildpipeline/alpine.3.6.groovy

```diff
@@ -24,10 +24,10 @@ simpleDockerNode('microsoft/dotnet-buildtools-prereqs:alpine-3.6-3148f11-2017111
         sh "./build-managed.sh -runtimeos=alpine.3.6 -- /t:GenerateVersionSourceFile /p:GenerateVersionSourceFile=true /p:PortableBuild=false"
     }
     stage ('Sync') {
-        sh "./sync.sh -p -runtimeos=alpine.3.6 -- /p:ArchGroup=x64 /p:PortableBuild=false"
+        sh "./sync.sh -p -runtimeos=alpine.3.6 -BuildTests=false -- /p:ArchGroup=x64 /p:PortableBuild=false"
```

Comment:
Is this a supported parameter to sync? If so is it needed?

Reply:
It is supported. We currently use it in the official builds. It ensures that the test runtime stuff will not get restored. I'd prefer using the same parameters on sync and build